packages/fluentui/react-context-selector/src/useContextSelector.ts

```diff
@@ -18,39 +12,70 @@ export const useContextSelector = <Value, SelectedValue>(
   context: Context<Value>,
```

Comment:
This shares quite a lot with `useContextSelectors` could we make this use the other hook and simply return a single value ? i.e.

```typescript
useContextSelector = (context, selector) => {
  const selectors = { single: selector };
  const {single}  = useContextSelectors(context, selectors)
  return single;
}
```

Reply:
not to mention that the plural version has a lot more readable conditions like `stateHasChanged` 😄

Reply:
I don't think that is the goal, WDYT @layershifter ?

Reply:
Let's keep these things separate now...